Source/WebKit/UIProcess/mac/WKTextFinderClient.mm

```diff
@@ -113,7 +113,7 @@ @interface WKTextFinderMatch : NSObject <NSTextFinderAsynchronousDocumentFindMat
 @end
 
 @implementation WKTextFinderMatch {
-    WKTextFinderClient *_client;
+    __weak WKTextFinderClient *_client;
     NSView *_view;
```

Comment:
Can we also clean this up while here?

Reply:
The `NSView`.

Reply:
Yep! (As we discussed in person, it _should_ really be a `RetainPtr`, but we'll deploy `__weak` here instead to maintain bincompat).